pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+**Informational** PHEPs are for the benefit of PyHC developers and users and may represent "best practices". They are not binding.
+
+**Process** PHEPs determine the process by which the PyHC community makes formal decisions and as such bind all participants in PyHC.
+
+For all three kinds, key words in PHEPs must be interpreted according to [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119); the use of `must`, `must not`, `should`, and `should not` are preferred to the alternatives.
+
+# PHEP Workflow
+## PHEP Editors
+<a name="phep-editors"></a>
+The PHEP editors are individuals responsible for managing the administrative and editorial aspects of the PHEP workflow (e.g. assigning PHEP numbers and changing their status). See [PHEP Editor Responsibilities & Workflow](#phep-editor-responsibilities-workflow) for details.
+
+PHEP editorship is by nomination of the current editors, subject to approval of PyHC leadership. In the event there are fewer than two active editors, PyHC leadership may appoint editors from active participants in the community, sufficient to bring the total to two.
+
+The editors can be contacted by mentioning `@heliophysicsPy/pep-editors` on GitHub. All of the PHEP workflow is conducted via the GitHub PyHC standards repository issues and pull requests.
+
+## Start with an idea
+The PHEP process begins with a new idea for PyHC. It is highly recommended that a single PHEP contain a single key proposal or new idea; the more focused the PHEP, the more successful it tends to be. The PHEP editors reserve the right to reject PHEP proposals if they appear too unfocused or too broad. If in doubt, split your PHEP into several well-focused ones.
+
+Each PHEP must have a champion - someone who writes the PHEP using the style and format described below, shepherds the discussions in the appropriate forums, and attempts to build community consensus around the idea. The PHEP champion (a.k.a. Author) should first attempt to ascertain whether the idea is PHEP-able. Posting to the PyHC mailing list or Element/Slack is usually the best way to go about this.
+
+Vetting an idea publicly before going as far as writing a PHEP is meant to save the potential author time. Asking the community first if an idea is original helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussions (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas.
```

Comment:
Would it make sense to suggest the creation of a GitHub issue in the repository?  That could become a place to discuss the idea in an open way, but it wouldn't need to be done this way.

Reply:
Issue for initial proposal, then PR with the PHEP that closes the issue? Intriguing....

Reply:
I feel like PyHC telecons are the most natural place to publicly propose ideas. But GitHub Issues for initial proposals would work fine too! I like that suggestion.

Reply:
See wording at https://github.com/jtniehof/standards/blob/4af2b621485d0ebb7ef4fb5aafb09544faae440a/pheps/phep-0001.md?plain=1#L61 and please "resolve" if happy

Reply:
I'm happy with the wording. Only reason I'm not "resolving" right away is because I'd personally make the hyphen after "Each PHEP must have an author" an em dash instead 🤓

Reply:
Hey, I gave you UTF-8 is _permitted_. Now you're going to push for _required_? ;)

(Actually, I think it should probably be a colon, while we're bikeshedding.)

Reply:
😂 Agree to meet in the middle with a colon? (100% bikeshedding)

Reply:
I jest, but this is a fairly large document...if tweaking punctuation improves readability, it's worth finding that. It'll be a colon in the next push.